src/Raven.Server/ServerWide/Commands/PeriodicBackup/UpdatePeriodicBackupStatusCommand.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace Raven.Server.ServerWide.Commands.PeriodicBackup
 {
+    // This command is for updating the cluster backup status only. We do not want this to touch the local backup status
```

Comment:
Let's make it simpler so we might not need to open a write transaction at all and update the `BackupStatusStorage` here.

Reply:
done. checking correct server by backup status' node tag